modules/virt-overriding-default-fs-overhead-value.adoc

```diff
@@ -33,16 +33,16 @@ spec:
       storageClass:
         <storage_class_name>: "<new_value_for_this_storage_class>" <2>
 ----
-<1> The file system overhead percentage that CDI uses across the cluster. For example, `global: "0.07"` reserves 7% of the PVC for file system overhead.
+<1> The default file system overhead percentage used for any storage classes that do not already have a set value. For example, `global: "0.07"` reserves 7% of the PVC for file system overhead.
 <2> The file system overhead percentage for the specified storage class. For example, `mystorageclass: "0.04"` changes the default overhead value for PVCs in the `mystorageclass` storage class to 4%.
 
-. Save and exit the editor to update the `CDI` object.
+. Save and exit the editor to update the `HCO` object.
 
 .Verification
 
-* View the `CDI` status and verify your changes by running the following command:
+* View the `CDIConfig` status and verify your changes by running the following command:
 +
 [source,terminal]
 ----
-$ oc get cdi -o yaml
+$ oc get cdiconfig -o yaml
```

Comment:
You can also more specifically verify your changes with:
oc get cdiconfig -o jsonpath='{.items..status.filesystemOverhead}'
The output looks like this:
{"global":"0.055","storageClass":{"csi-manila-ceph":"0.055","hostpath-csi-basic":"0.055","hostpath-csi-pvc-block":"0.055","local-block-hpp":"0.055","local-block-ocs":"0.055","nfs":"0.055","ocs-storagecluster-ceph-rbd":"0.055","ocs-storagecluster-ceph-rgw":"0.055","ocs-storagecluster-cephfs":"0.055","standard-csi":"0.055"}}